portfolio/src/main/webapp/style.css

```diff
@@ -1,15 +1,32 @@
+.title {
```

Comment:
nit: Please change back to the two space indent style for the file

https://google.github.io/styleguide/htmlcssguide.html#Indentation